ruby/basics/arrays.md

```diff
@@ -1,3 +1,18 @@
+<!--Chris Pine covers:
+* array literals
+* accessing via index
+* #each & #times methods
+* #to_s, #join
+* #push/#pop, #last, #length-->
+
+<!--I add:
+* array initializing
+* negative indices
+* shift/unshift
+* concatenating/substracting
+* set operations?
+* other methods-->
+
 # Arrays
 At the beginning of this section, you learned about creating and manipulating individual instances of number and string objects. In real-world development, where you'll be working with dozens (and even hundreds!) of such objects and variables, working with individual instances will be tedious, if not impossible.
```

Comment:
Can we reword this a little. Theres a lot of mention of `objects` which is of course completely correct. I think it might be difficult for a user to understand what an object is at this early point due to how abstract it is. I think that using concrete objects might be better, eg "strings, numbers" etc